code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -301,7 +301,7 @@
 		if(ranged_message)
 			visible_message("<span class='warning'><b>[src]</b> [ranged_message] at [target]!</span>", 1)
 		if(ckey)
-			add_attacklogs(src, target, "[ranged_message ? ranged_message : "shot"] at")
+			add_attacklogs(src, target, "[ranged_message ? ranged_message : "shot"] at", admin_warn = FALSE)
```

Comment:
Why false?

Reply:
I don't know how useful reporting every bullet to admins is, especially when this kind of thing is usually from a bus. But I can change it back, sure.

Reply:
It'd be for the best.  We can't predict all the times a player shooting another player will be good or bad.